elements/rh-footer/rh-footer-lightdom.css

```diff
@@ -1,40 +1,42 @@
-rh-footer :is([slot^=links]) a {
-  gap: var(--rh-footer-links-gap, 10px);
+/* ensure links fully wrap img tags */
+:is(rh-footer, rh-global-footer) a[slot^=logo] {
+  display: block;
+}
+
+rh-footer [slot^=links] a {
+  gap: var(--rh-footer-links-gap, 0.625em);
```

Comment:
I'm not overlay thrilled about setting gap / padding / margins to `ems`.  Seems like it would make it more difficult to translate the design specs to code. 🤷‍♂️